src/wait.js

```diff
@@ -0,0 +1,3 @@
+export default async function wait(interval) {
+	return new Promise((resolve) => setTimeout(resolve, interval));
```

Comment:
I'd rather this inline instead of as a separate file, since it's a one liner and it's only used in one place.